### PR TITLE
feat: typography

### PR DESCRIPTION
**Pull request by Heavenlydawn**
Branch Name and Commit Message Changed

### CODE REVIEW

File: src/app/[locale]/globals.css [hunk 1]
```diff
@@ -7,7 +7,89 @@
...
 	}
 
+	/* 
+	===============
+	Extra Heading 
+	===============
+	*/
 	.vm-text-eh1 {
 		@apply text-6xl font-semibold leading-[70px] tracking-tight;
 	}
```
Comment (zorro-ftw) [inline src/app/[locale]/globals.css:17]: Using arbitrary values in Tailwind is a no-no. They are a huge performance killer. So, please try to make use of arbitrary values as less as possible, ideally never. Don't forget, these classes you write here, will be applied to all texts in this project. You don't need everything to be pixel-perfect if you have close enough values.

In the names of Typographies, our design has the ratio of line heights. The following is the name for extra heading 1 and you can easily see that it has 120% line height.

`Extra Heading/Eh1 - 58 Semi Bold -  Lh 120%`

From the following link, you can see that Tailwind has some ratio-based line height setters like `leading-none`, `leading-tight` etc. Try to use closer ratios from built-in Tailwind classes. Please apply this change to all the typography classes.

File: src/app/[locale]/globals.css [hunk 2]
```diff
@@ -7,7 +7,89 @@
...
+	.vm-text-eh2 {
+		@apply text-4xl font-semibold leading-[53.2px] tracking-tight;
+	}
+	.vm-text-eh3 {
+		@apply text-3xl font-semibold leading-[44.8px] tracking-tight;
+	}
+	.vm-text-eh4 {
+		@apply text-[28px] font-semibold leading-[39.2px] tracking-tight;
```
Comment (zorro-ftw) [inline src/app/[locale]/globals.css:25]: No need to be pixel perfect. Let's go for text-3xl for this one as well.

```suggestion
		@apply text-3xl font-semibold leading-[39.2px] tracking-tight;
```

File: src/app/[locale]/globals.css [hunk 3]
```diff
@@ -7,7 +7,89 @@
...
+		@apply text-2xl font-semibold leading-[33.6px] tracking-tight;
+	}
+
+	/* 
+	===============
+	Heading
+	===============
+	*/
+	.vm-text-h1 {
+		@apply text-[34px] font-semibold leading-[40.8px]  tracking-tight;
```
Comment (zorro-ftw) [inline src/app/[locale]/globals.css:40]:
```suggestion
		@apply text-4xl font-semibold leading-[40.8px]  tracking-tight;
```

**Review by zorro-ftw — CHANGES_REQUESTED** [review]

File: src/app/[locale]/globals.css (hunk 1 above)
Comment (zorro-ftw) [reply src/app/[locale]/globals.css:17]: Here is the link btw

https://tailwindcss.com/docs/line-height

Comment (Heavenlydawn) [reply src/app/[locale]/globals.css:17]: Noted I will make the necessary corrections 
I will take out the arbitrary values and use the ratio based line height instead

Comment (Heavenlydawn) [reply src/app/[locale]/globals.css:16]: I have made the necessary changes

**Review by zorro-ftw — APPROVED** [review]

**Pull request merged**